lib/mayaUsd/ufe/UsdUndoMaterialCommands.cpp

```diff
@@ -157,10 +157,32 @@ UsdUndoAssignNewMaterialCommand::UsdUndoAssignNewMaterialCommand(
     const UsdSceneItem::Ptr& parentItem,
     const std::string&       nodeId)
     : Ufe::InsertChildCommand()
-    , _parentPath(parentItem->path())
     , _nodeId(nodeId)
     , _cmds(std::make_shared<Ufe::CompositeUndoableCommand>())
 {
+    if (!parentItem || !parentItem->prim().IsActive())
+        return;
+
+    Ufe::Path parentPath        = parentItem->path();
+    const UsdStageWeakPtr stage = getStage(parentPath);
+    _stagesAndPaths[stage].push_back(parentPath);
+}
+
+UsdUndoAssignNewMaterialCommand::UsdUndoAssignNewMaterialCommand(
```

Comment:
Does that code need `UFE_PREVIEW_XX` defines protection?

Reply:
Good question, honestly not sure. 

These changes are entirely contained in MayaUSD, with no UFE update required. What UFE version would I be looking for?